packages/server/modules/core/repositories/users.ts

```diff
@@ -6,6 +6,9 @@ import { metaHelpers } from '@/modules/core/helpers/meta'
 import { UserValidationError } from '@/modules/core/errors/user'
 import { Knex } from 'knex'
 import { Roles, ServerRoles } from '@speckle/shared'
+import { updateUserEmailFactory } from '@/modules/core/repositories/userEmails'
+import knexInstance from '@/db/knex'
```

Comment:
No big deal, but i'd just name the import `knex` or even `db` ([to support object property shorthand syntax in factories](https://gomakethings.com/object-property-shorthand-values-with-vanilla-javascript/)), the `Instance` suffix is redundant considering that the difference between the instance and the knex factory is only relevant within `@/db/knex`

Reply:
@alemagio Is this called `knexInstance` because of VSCode auto-imports, not you manually typing it out that way? If so, I suggest changing the name of the `knexInstance` const inside of `@/db/knex.js` to what we want the import to be called, cause that's what I think VSCode resolves the auto-import name from. Maybe `const db`? So that auto-import always does `import db from '@/db/knex'`

Reply:
yes, that's the auto import.
To avoid breaking existing code I will simply add another export so from now on we can simply have
```ts
import { db } from '...'
```

Reply:
@alemagio Breaking how? The default export can have any kind of arbitrary name, changing the const inside of knex.js only will change VSCode's mechanism for figuring out what to call it, all other places that still do `import whateverArbitraryName from '@/db/knex'` will still work as before

Reply:
![image](https://github.com/user-attachments/assets/1f1ed99a-0137-453d-80b6-2cfd177525cf)

Reply:
oh you meant renaming it in the import 
I thought you meant in the export.
Anyway I think I found a good compromise